cli/prompt.go

```diff
@@ -164,7 +164,13 @@ func (*Handler) PromptInt(
 }
 
 func (*Handler) PromptChoice(label string, max int) (int, error) {
-	// TODO: auto-select if only 1 option provided
+	// auto-select if only 1 option provided
+	if max == 1 {
+		utils.Outf(
+			"{{cyan}}only one chain available:{{/}} autoselect\n",
```

Comment:
This can be used for more than just "chains". Here are some more examples of usage: 
<img width="724" alt="image" src="https://github.com/ava-labs/hypersdk/assets/13023275/60ddc2a3-d04f-40ec-beba-56de489803d2">

Reply:
I think we should print the "label" concatenated with `0` (ideally in a similar form as `promptui`)